static/js/NavSidebar.jsx

```diff
@@ -85,6 +87,59 @@ const TitledText = ({enTitle, heTitle, enText, heText}) => {
   </Module>
 };
 
+const RecentlyViewed = ({toggleSignUpModal, mobile}) => {
+   const [recentlyViewedItems, setRecentlyViewedItems] = useState(null);
+   const handleAllHistory = (e) => {
+    if (!Sefaria._uid) {
+      e.preventDefault();
+      toggleSignUpModal(SignUpModalKind.ViewHistory);
+    }
+    gtag('event', 'recently_viewed', {link_type: 'all_history', logged_in: !!Sefaria._uid});
+   }
+   const trackItem = (ref, pos) => {
```

Comment:
Whats the relationship to the tracking code above this? 
i.e. why are there two?

Reply:
trackItem tracks each ref in the list, whereas the gtag in handleAllHistory tracks clicks of the "All History" link.